### PR TITLE
Added UTF-8 enconding compliance check for submitted .md files.

### PR DESCRIPTION
**Pull request by mpinheir**
Added UTF-8 enconding compliance check for submitted .md files.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@ install:
 script:
   - travis/checkparticipantes PARTICIPANTES.md
   - travis/hugo -v
+  - utfcheck.sh
```
Comment (marcopaganini) [inline .travis.yml:19]: Use travis/utfcheck.sh

File: travis/utfcheck.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```
Comment (marcopaganini) [inline travis/utfcheck.sh:1]: Maybe add a comment starting on line 2 explaining what this script does (look at the similar script in op-desafios, but adapt the comments to this case.)

**Review by marcopaganini — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (mpinheir) [reply .travis.yml]: Done

File: travis/utfcheck.sh (hunk 2 above)
Comment (mpinheir) [reply travis/utfcheck.sh:1]: Done

**Review by marcopaganini — APPROVED** [review]

**Pull request merged**